tests/Functional/ContentRenderTestBase.php

```diff
@@ -205,4 +206,118 @@ protected function createContactEntity(string $name, string $bundle, int $status
     return $contact;
   }
 
+  /**
+   * Asserts Contact entity field rendering.
+   *
+   * @param \Behat\Mink\Element\NodeElement $element
+   *   Rendered element.
+   * @param string $name
+   *   Name of the Contact entity.
+   */
+  protected function assertContactEntityDefaultDisplay(NodeElement $element, string $name):void {
+    $contact_name = $element->findAll('css', 'h3');
+    $this->assertCount(1, $contact_name);
+    $this->assertEquals($name, $contact_name[0]->getText());
+
+    $contact_body = $element->findAll('css', '.ecl-editor');
+    $this->assertCount(1, $contact_body);
+    $this->assertEquals("Body text $name", $contact_body[0]->getText());
```

Comment:
If there is no image the text of the body should be corresponding to the width of the body, right?
![image](https://user-images.githubusercontent.com/22004498/98805267-99e98d80-2417-11eb-8c19-156694d13379.png)